resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -193,6 +193,7 @@ cloudformation.yaml.too_many_documents=There should only be 1 YAML document per
 cloudformation.yaml.too_many_files=Found {0} YAML files but only expected 1
 cloudwatch.logs.compare.start.end.date=Start date must be before end date
 cloudwatch.logs.copy_action=Copy {0, choice, 1#message|2#messages}
+cloudwatch.logs.default_query=fields @timestamp, @message| sort @timestamp desc| limit 20
```

Comment:
This should not be in this file as it's not something we can localize, assign it to a `const val` somewhere

Reply:
Added the default query as const val